projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/RunJob.tsx

```diff
@@ -123,7 +123,23 @@ export const findFailingCellId = (message: String): string => {
   return '';
 };
 
+export const getCellInputAreaPromp = (failinCell: Element): Element | undefined => {
```

Comment:
nit: may you document the purpose of this function? It's not very clear when reading the definition.

Reply:
done